package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@feathery/react",
-  "version": "1.1.9",
+  "name": "@feathery-org/staging",
+  "version": "1.1.36",
```

Comment:
should probably use alpha/beta tags in the future for something like this. wasn't too concerned since we don't need to use the private package very often